Framework/Kernel/inc/MantidKernel/Material.h

```diff
@@ -196,6 +199,8 @@ class MANTID_KERNEL_DLL Material final {
   void saveNexus(::NeXus::File *file, const std::string &group) const;
   void loadNexus(::NeXus::File *file, const std::string &group);
 
+  bool hasValidxRayAttenuationProfile();
```

Comment:
This is a bit pedantic but given you'll probably be doing a few other changes hopefully no big deal. Please can you change to hasValidXRayAttenuationProfile?